backend/dbscripts/runtimedb/sqlite.sql

```diff
@@ -36,6 +36,7 @@ CREATE TABLE FLOW_CONTEXT (
     FLOW_ID VARCHAR(36) NOT NULL,
     DEPLOYMENT_ID VARCHAR(255) NOT NULL,
     APP_ID VARCHAR(36) NOT NULL,
+    VERBOSE INTEGER NOT NULL DEFAULT 0,
```

Comment:
Do we need to keep it as a first class attribute in the table?  can't we have it under RUNTIME_DATA

Reply:
I'd like to keep it as a separate top level field as once send, it's applied during the while flow similar to application id or flow type.
However we can store JSON data in DB, but current DB layout only stores context runtime data in `RUNTIME_DATA` column. So we need to have a separate column until a refactoring is done